### ISSUE TITLE
tCore bundled resource update script fails

### ISSUE THREAD

**Issue by PhotoNomad0** [unfoldingWord/translationCore#6062]
### User Story

When I run script `npm run update-resources` to update the resources bundled in the tCore app, it replaces all books.zip files that have NOT been updated with an empty books.zip.

### Features / Specifications

- [ ]
- [ ]
- [ ]

### Definition of Done

- [ ]
- [ ]
- [ ]

## Additional Context

This is a bug in 1.1.4 and 1.2.0.  The current work-around is to check each bible.zip.  Any bible.zip files that are small (about 22 bytes) will either need to be restored from git or regenerated by deleting the bible folder and rerunning the script.  Another work-around in 1.2.0 is to just run `overwrite-resources` and regenerate everything (unfortunately this crashes in 1.1.4).

Probably only needs to be fixed for 1.2.0 release as we have already worked around it for 1.1.4.

**Comment by cckozie** [comment]
This is not working in 1.2.0 (142b2c9) 
I deleted the resources folder, then started tC. Not seeing the Hindi tA folder.
The Hindi tA is not available until after downloading Hindi from the source content update

**Comment by PhotoNomad0** [comment]
@cckozie - sorry, didn't communicate well.  This is not a fix for the tCore resources folder.  This is a fix to a script we run to update the "bundled" resources before release.  I don't know if there is an easy way for you to directly test this.  This is for manual bundled resource updates and also in preparation for automatic updates in release builds.

As far as Hindi TA not in develop branch, we were waiting to update the bundled resources in develop until we merged latest changes from 1.1.4 into develop.  I just added a new fix for 1.1.4 that needs to be into develop before we can update the bundled resources.
